pkgs/development/python-modules/scipy/default.nix

```diff
@@ -35,11 +52,9 @@ buildPythonPackage rec {
   enableParallelBuilding = true;
 
   checkPhase = ''
-    runHook preCheck
```

Comment:
This silently breaks `preCheck` and `postCheck` for users trying to use overrides.

Reply:
Yeah so question about this. Isn't this just the way it has always been done for ALL python packages? I looked at the got blame and those lines were written 6 years ago.

Reply:
I don't know the answer to your question. However, I did check that this breaks hooks even for python packages.